allauth/socialaccount/providers/metamask/templates/metamask/auth.html

```diff
@@ -0,0 +1,73 @@
+{% load static %}
+<div id="personalSign">  </div>
+<HR>
+<div id="personalSignResult">  </div>
+<HR>
+<script type="text/javascript">
+if (typeof jQuery == 'undefined') {
```

Comment:
I would really prefer to avoid a jQuery dependency. Just use `window.fetch()`,

Reply:
OK - I am so used to using jQuery I didn't think about it. On the list to fix tonight.

Reply:
Fixed this!

Reply:
My latest update changes to fetch.